CHANGELOG.md

```diff
@@ -1,5 +1,114 @@
+* Added the ability to open a file by specifying its name in the terminal (Linux)
+* Updated images for the error pages
+* Added the translation into Serbian (sr-Latn-RS, Serbian (Latin, Serbia and Montenegro))
+* Added a button to select a local file in the hyperlink insertion dialog window
+* Changes in program interface: manageable functional buttons, Replace button is
+  now on the Home tab, Copy style, Clear style, Select all
+* Autoshape shadowing settings
+* Updated the set of color themes available in editors
+* Added internal help in Portuguese (pt-BR)
+* Added Arabic to all regional settings
+* Added a title and a button to close the panel to the `Chat` panel
+
+#### Document Editor
+
+* New button on the top toolbar for changing document editing mode: `Editing`,
+  `Reviewing`, `Viewing`
+* Added tooltips for new or updated functionality (displayed when loading the
+  editor or when switching to the corresponding tab)
+* Implemented the ability to set the format for page numbering
+* Added support for the page color
+* Updated built-in paragraph styles
+* New items in the indents menu for opening the right panel and managing paragraph
+  indents, the ability to manage paragraph indents via the top toolbar
+* Color theme button is now on the Layout tab
+* Mail merge button is now on the Collaboration tab
+* Line spacing options updated
+* Improved work of the algorithm for displaying numbers and punctuation in
+  Arabic text for the Neutral and Weak classes
+* Improved fitting for paragraphs with main `RTL` direction
+
+#### Spreadsheet Editor
+
+* New languages added: ligature support
+* Selected cells are highlighted on their respective row/column numbers
+* New functions: `GETPIVOTDATA`, `IMPORTRANGE`
+* New function category: Custom based on jsdoc
+* Version history update: edited cells are highlighted
+* Users get custom protected range cells viewing rights
+* Implemented the ability to copy/move sheets between opened books
+* Changed the appearance of the sheet list in the embedded viewer in accordance
+  with the styles of the main spreadsheet editor
+
+#### Presentation Editor
+
+* New slide settings on the right panel: show background graphics, reset background
+  to the theme background, apply settings to all slides
+* Added `Animation` pane
+* Added a mode for editing master slides and templates
+* Updated the player interface for playing audio/video files (Windows and Linux)
+
+#### Forms
+
+* When adding a fixed form, now it is inserted without wrapping in front of the text
+* The color of the  Picture placeholder corresponds to the color of the role
+  for this form
+* The thickness of the frame for required fields is now 2 px with any zoom
+* Added a button to switch to the editing mode (similar to the button in
+  the editor header) for forms opened in the View or Fill forms mode
+* The presence of this button depends on the integrator\'s subscription to
```

Comment:
This is optional, by default the buttons are always displayed, so this line can be deleted